test/travis-before-install.sh

```diff
@@ -1,16 +1,30 @@
 #!/bin/bash
 set -o xtrace
 
-# Travis does shallow clones, so there is no master branch present.
-# But test-no-outdated-migrations.sh needs to check diffs against master.
-# Fetch just the master branch from origin.
-( git fetch origin master
-git branch master FETCH_HEAD ) &
-# Github-PR-Status secret
-if [ -n "$encrypted_53b2630f0fb4_key" ]; then
-  openssl aes-256-cbc \
-    -K $encrypted_53b2630f0fb4_key -iv $encrypted_53b2630f0fb4_iv \
-    -in test/github-secret.json.enc -out /tmp/github-secret.json -d
+if [ "${TRAVIS}" == "true" ]; then
+  # Boulder consists of multiple Go packages, which
+  # refer to each other by their absolute GitHub path,
+  # e.g. github.com/letsencrypt/boulder/analysis. That means, by default, if
+  # someone forks the repo, Travis won't pass on their own repo. To fix that,
+  # we add a symlink.
+  mkdir -p $TRAVIS_BUILD_DIR $GOPATH/src/github.com/letsencrypt
+  if [ ! -d $GOPATH/src/github.com/letsencrypt/boulder ] ; then
+    ln -s $TRAVIS_BUILD_DIR $GOPATH/src/github.com/letsencrypt/boulder
+  fi
+
+  # Travis does shallow clones, so there is no master branch present.
+  # But test-no-outdated-migrations.sh needs to check diffs against master.
+  # Fetch just the master branch from origin.
+  ( git fetch origin master
+  git branch master FETCH_HEAD ) &
```

Comment:
Unrelated to this branch, but I'd like to circle back and figure out why we background this fetch and branch call (because I'm worried it'll race with whatever needs it).